src/arload.py

```diff
@@ -123,6 +125,8 @@ def optarg(argflag, argv, pypname):
         if o in ('-h', '--help'): msgs.usage(None)
         elif o in ('-c', '--cpus'): argflag['run']['ncpus'] = a
         elif o in ('-v', '--verbose'): argflag['out']['verbose'] = int(a)
+    for a in arg:
```

Comment:
I’ve decided not to implement this for now.  Realized that we’d
need to input df values and probably have to do something rather 
more complex than curvefit.  Will work on a Moffat profile instead.



> On Apr 14, 2016, at 4:16 PM, Ryan Cooke <notifications@github.com> wrote:
> 
> In src/arload.py <https://github.com/PYPIT/PYPIT/pull/40#discussion_r59807503>:
> 
> > @@ -123,6 +125,8 @@ def optarg(argflag, argv, pypname):
> >          if o in ('-h', '--help'): msgs.usage(None)
> >          elif o in ('-c', '--cpus'): argflag['run']['ncpus'] = a
> >          elif o in ('-v', '--verbose'): argflag['out']['verbose'] = int(a)
> > +    for a in arg:
> You should feed the flags first, like this:
> 
> pypit --use_masters --help lris_b_ls_600_4000_d560.red
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/PYPIT/PYPIT/pull/40/files/32182584b47d012ba1ad5056bf0e954e67b9c7af#r59807503>
> !DSPAM:826,571024c2150421203214212!

----------------------------------------------
J. Xavier Prochaska
UCO/Lick Observatory
1156 High St.
UC Santa Cruz
Santa Cruz, CA 95064

http://www.ucolick.org/~xavier/
http://imps.ucolick.org/
http://www.qpqsurvey.org/
http://k1dm3.ucolick.org/
http://www.ucolick.org/~xavier/IDL/index.html
https://github.com/pyigm/pyigm
https://github.com/linetools/linetools

831-459-2135 (Direct)
831-459-2991 (UCO/Lick Main)
831-459-5244 (Fax)